requirements-dev.txt

```diff
@@ -10,7 +10,7 @@
 -r tools/c7n_mailer/requirements.txt
 -r tools/c7n_org/requirements.txt
 -r tools/c7n_terraform/requirements.txt
--r tools/c7n_tencentcloud/requirements.txt
+# -r tools/c7n_tencentcloud/requirements.txt
```

Comment:
this looks odd?

Reply:
Right, I switched off for testing locally. Sorry. I reverted the code.